site/src/main/scala/scalatex/site/Site.scala

```diff
@@ -84,7 +82,7 @@ trait Site{
   /**
    * The contents of this page
    */
-  def content: Map[String, Frag]
+  def content: Map[String, (Seq[Frag], Frag)]
```

Comment:
This definitely needs a comment explaining what this odd data-structure is. Even the old comment is out of data (we can have more than one page) so you'll need to update it